App.js

```diff
@@ -0,0 +1,146 @@
+import React, {Fragment} from 'react';
+import {StyleSheet, Text} from 'react-native';
+import * as firebase from 'firebase';
+import * as Facebook from 'expo-facebook';
+
+import Home from './home';
+
+// Initialize Firebase
+const firebaseConfig = {
+  apiKey: 'AIzaSyBbs01R7oXNzuVjhtVCpexnn2mjQuYV-zU',
+  authDomain: 'shenkar-games.firebaseapp.com',
+  projectId: 'shenkar-games',
+  storageBucket: 'shenkar-games.appspot.com',
+};
+!firebase.apps.length ? firebase.initializeApp(firebaseConfig) : firebase.app();
+
+import {Container, Form, Input, Item, Button, Label} from 'native-base';
+
+export default class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.signUpUser = this.signUpUser.bind(this);
+    this.loginUser = this.loginUser.bind(this);
+    this.loginWithFacebook = this.loginWithFacebook.bind(this);
+    this.state = {
+      email: '',
+      password: '',
+      isLogged: false,
+    };
+  }
+
+  componentDidMount() {
+    firebase.auth().onAuthStateChanged((user) => {
+      if (user != null) {
+        console.log(user);
+      }
+    });
+  }
+
+  signUpUser(email, password) {
+    try {
+      if (this.state.password.length < 6) {
+        alert('Please enter atleast 6 characters');
+        return;
+      }
+
+      firebase
+        .auth()
+        .createUserWithEmailAndPassword(email, password)
+        .then(this.setState({isLogged: true}));
+    } catch (error) {
+      console.log(error.toString());
+    }
+  }
+  loginUser(email, password) {
+    try {
+      firebase
+        .auth()
+        .signInWithEmailAndPassword(email, password)
+        .then(this.setState({isLogged: true}))
+        .catch((error) => {
+          console.log(error);
+        });
+    } catch (error) {
+      console.log(error.toString());
+    }
+  }
+  async loginWithFacebook() {
+    await Facebook.initializeAsync({
+      appId: '878071626358300',
```

Comment:
give everyone in our group access to the app 👍 
https://docs.google.com/spreadsheets/d/1SM0LuJcHDhfGaGYhR3-ibPqhJ3x9y1gPkjpMVqCyhrQ/edit#gid=860237573